modules/virt-template-vmi-probe-config.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-monitoring-vm-health.adoc
+
+[id="virt-template-vmi-probe-config_{context}"]
+= Template: Virtual machine instance configuration file for defining health checks
+
+[source,yaml]
+----
+  apiVersion: kubevirt.io/v1alpha3
+  kind: VirtualMachineInstance
+  metadata:
+    labels:
+      special: vmi-fedora
+    name: vmi-fedora
+  spec:
+    domain:
+      devices:
+        disks:
+        - disk:
+            bus: virtio
+          name: containerdisk
+        - disk:
+            bus: virtio
+          name: cloudinitdisk
+      resources:
+        requests:
+          memory: 1024M
+    readinessProbe:
+      httpGet:
+        port: 1500
+      initialDelaySeconds: 120
+      periodSeconds: 20
+      timeoutSeconds: 10
+      failureThreshold: 3
+      successThreshold: 3
+    terminationGracePeriodSeconds: 0
+    volumes:
+    - name: containerdisk
+      registryDisk:
+        image: kubevirt/fedora-cloud-registry-disk-demo
+    - cloudInitNoCloud:
+        userData: |-
+          #cloud-config
+          password: fedora
+          chpasswd: { expire: False }
+          bootcmd:
+            - setenforce 0
+            - dnf install -y nmap-ncat
+            - systemd-run --unit=httpserver nc -klp 1500 -e '/usr/bin/echo -e HTTP/1.1 200 OK\\n\\nHello World!'
```

Comment:
When running the VM the probe fails with
```
Readiness probe failed: Get "http://10.129.2.34:1500/": net/http: HTTP/1.x transport connection broken: unexpected EOF
```
Going into the VM and using curl the connection looks fine.
Maybe @phoracek knows if the HTTP probe is right this way?

Reply:
Could it be that firewall is blocking that port? Does calling curl from the outside get to the VM?

Reply:
I tried again today and it was fine. Might have been that or just my impatience.